components/HomePage/query.graphql

```diff
@@ -0,0 +1,8 @@
+query {
```

Comment:
In order to use graphql files in the client we're going to have to add a third party library. I've been using `graphql.macro' with CRA but I'm pretty sure we'll have to find another library for next.

Reply:
👍 placeholder just to get the idea across.